src/app/Fake.Core.Trace/Trace.fs

```diff
@@ -44,12 +44,24 @@ let traceVerbose s =
 /// Writes a trace to stderr (in yellow)  
 let traceImportant text = CoreTracing.postMessage (TraceData.ImportantMessage text)
 
+/// Writes a message to stderr (in yellow) 
+let traceImportantfn fmt = Printf.ksprintf traceImportant fmt
+
+/// Writes a message to stderr (in yellow) and without a line break
+let traceImportantf fmt = Printf.ksprintf (fun text -> CoreTracing.postMessage (TraceData.ImportantMessage(text))) fmt
+
 /// Writes a trace to the command line (in yellow)
 let traceFAKE fmt = Printf.ksprintf (TraceData.ImportantMessage >> CoreTracing.postMessage) fmt
 
 /// Traces an error (in red)
 let traceError error = CoreTracing.postMessage (TraceData.ErrorMessage error)
 
+/// Writes an error message to stderr (in red) 
+let traceErrorfn fmt = Printf.ksprintf traceError fmt
+
+/// Writes an error message to stderr (in red) and without a line break
+let traceErrorf fmt = Printf.ksprintf (fun text -> CoreTracing.postMessage (TraceData.ErrorMessage(text))) fmt
```

Comment:
I copied the method from other f variants...happy to remove them though, I don't get the use case of the f methods

Reply:
It is the `, false` parameter - which doesn't exist for those types for some reason.

>  I don't get the use case of the f methods

One use case is to realise some prompt to the user like
`Please enter value for X: |` and have the cursor sitting at `|` instead of the next line

Reply:
Oh now it makes perfect sense!
I can't think why someone would need important or error version anyways then.  So I'll remove.